atlp-rwanda/rca-barefoot-team3-fe
# Ft configure storybook #184637942

## What does this PR do?

Configures storybook for component development

## How should this be manually tested?

In project root run **yarn run storybook**
you will be able to see the interface in the browser

## Any background context you want to provide?

No

## What are the relevant pivotal tracker/Trello stories?

Configure storybook for component development (#184637942)

## Screenshots (if appropriate)
<img width="951" alt="image" src="https://user-images.githubusercontent.com/53978074/233191036-c4ec7fcf-b37d-41a2-95e1-c5691d4331bf.png">
